dapps/digital-twin-detail/src/i18n/de.json

```
@@ -0,0 +1,34 @@
+{
+  "_twin-detail": {    
+    "data": {
+      "data-title": "Daten",
+      "general": {
+        "desc": "Beschreibung",
+        "general-title": "Allgemein",
+        "information-title": "Informationen",
+        "name": "Name",
+        "owner": "Besitzer",
+        "twin-image": "Zwilling Bild",
```

this or `Zwillingsbild`

```suggestion
        "twin-image": "Bild",
```